CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ Running
 ------------
 ``` bash
 # Start communication gateway
-docker-compose up gateway
+docker-compose up broker
```

Comment:
Are we considering to officially name it broker? :)

Reply:
Yes, such recommendation was in the air